plugin.video.mediathekview/addon.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
suggestion: spdx based headers at some point. Those type of license headers are far too big.

Reply:
Done